src/emailagent.cpp

```diff
@@ -669,24 +669,31 @@ void EmailAgent::expungeMessages(const QMailMessageIdList &ids)
     exportUpdates(accountList);
 }
 
-void EmailAgent::downloadAttachment(int messageId, const QString &attachmentlocation)
+void EmailAgent::downloadAttachment(int messageId, const QString &attachmentLocation)
 {
     m_messageId = QMailMessageId(messageId);
     QMailMessage message (m_messageId);
-
+    QMailMessagePart attachmentPart;
+    bool partFound = false;
```

Comment:
Why isn't this just something like "attachmentPart = message.partAt(QMailMessage::Location(attachmentLocation));" ?